builtins/amp-img.js

```diff
@@ -45,8 +48,15 @@ export class AmpImg extends BaseElement {
     /** @private {?UnlistenDef} */
     this.unlistenLoad_ = null;
 
+    /** @private @const {boolean} */
+    this.useNativeSrcset_ = isExperimentOn(this.win, 'amp-img-native-srcset');
+
+    /** @private @const {boolean} */
+    this.expOn_ = isExperimentOn(this.win, 'amp-img-blurry-placeholder');
```

Comment:
No need to check this yet.

Reply:
Sorry for the confusion, but by "check this" I'm assuming you mean I don't have to initialize this yet. If I'm wrong, again really sorry, just please let me know